ui/app/components/job-editor.js

```diff
@@ -37,11 +38,10 @@ export default class JobEditor extends Component {
     }
 
     if (this.args.variables) {
+      console.log('aye vars', this.args.variables);
```

Comment:
Is this expected to be here, or remains of debugging?

Reply:
hah, remains of debugging. Thanks for spotting it!